pep-0458-tuf-online-keys.rst

```diff
@@ -395,6 +395,208 @@ specifying `delegations`__.
 __ https://github.com/theupdateframework/tuf/tree/develop/tuf#delegations
 
 
+File Formats of the PyPI JSON Metadata
+--------------------------------------
+
+This section presents the format of the JSON metadata files.  Examples of the
+roles and their formats mentioned here are available for review in the
```

Comment:
"and their formats mentioned here" seems a bit off. Can you just delete "mentioned here"?